LICENSE.txt

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2024-present Isaac To <candleindark@gmail.com>
```

Comment:
```suggestion
Copyright (c) 2024 Isaac To <candleindark@gmail.com>
```
since eventually might/would become not correct , so best to mention real year ranges